src/server.ts

```diff
@@ -119,9 +123,9 @@ const startServer = async () => {
     server.post('/login-org', routeLoginOrg)
     server.post('/create-hash', routeCreateHash)
     server.post('/generate-pdf', routeGeneratePDF)
-    server.get('/outcomes', routeOutcomes)
-    server.get('/outcomes/table/:tableName', routeOutcomesTable)
-    server.get('/outcomes/table/:tableName/item/:id', routeOutcomesDetail)
+    server.get('/data-displays', routeDataDisplays)
+    server.get('/data-displays/table/:tableName', routeDataDisplayTable)
+    server.get('/data-displays/table/:tableName/item/:id', routeDataDisplayDetail)
```

Comment:
I'm confused. The front-end is using `data` and it works! But here is clearly set as `data-displays` 😮

Reply:
Ok - confusion sorted. I see that Front-end is setting the route as `data` and this is just to get the definition to how the data gets displayed. 👍